kythe/typescript/indexer.ts

```diff
@@ -1106,7 +1117,7 @@ class Visitor {
     if (vname && decl.kind === ts.SyntaxKind.PropertyDeclaration) {
       const declNode = decl as ts.PropertyDeclaration;
       if (this.isStaticMember(declNode, declNode.parent)) {
-        this.emitFact(vname, 'tag/static', '');
+        this.emitFact(vname, '/kythe/tag/static' as FactName, '');
```

Comment:
Should this be in kythe.ts?

Reply:
This is a non-standard Kythe fact name. We can add it though.